Plugin.php

```diff
@@ -87,4 +92,29 @@ public function bootPackages()
             }
         }
     }
+
+    public function enableJavascriptSentry(?string $javascriptDsn)
+    {
+        $script = Url::asset('/plugins/winter/sentry/assets/js/dist/sentry.js');
+        $env = Config::get('app.env');
+        $sampleRate = Config::get('sentry.traces_sample_rate');
+
+        $html = <<<HTML
+            <script>
+                window.sentryEnv = window.sentryEnv || {};
+                window.sentryEnv.SENTRY_JAVASCRIPT_DSN = "{$javascriptDsn}";
+                window.sentryEnv.APP_ENV = "{$env}";
+                window.sentryEnv.SENTRY_TRACES_SAMPLE_RATE = {$sampleRate};
+            </script>
+            <script src="{$script}"></script>
+        HTML;
+
+        Event::listen('backend.layout.extendHead', function () use ($html) {
+            return $html;
+        });
+
+        Event::listen('cms.page.render', function ($controller, string $pageContents) use ($html) {
+            return $pageContents . $html;
+        });
+    }
```

Comment:
You should be able to use the `Block::append('scripts', $generatedScripts);` method here to inject into the `{% scripts %}` tag in the theme. I think you could even do the following:
```suggestion
        $html = <<<HTML
            <script>
                window.sentryEnv = window.sentryEnv || {};
                window.sentryEnv.SENTRY_JAVASCRIPT_DSN = "{$javascriptDsn}";
                window.sentryEnv.APP_ENV = "{$env}";
                window.sentryEnv.SENTRY_TRACES_SAMPLE_RATE = {$sampleRate};
            </script>
            <script src="{$script}"></script>
        HTML;

        if ($this->app->runningInBackend()) {
            // Inject into the Block placeholder `head` in the backend
            Block::append('scripts');
        } elseif (!$this->app->runningInConsole()) {
            // Inject into the {% scripts %} tag in the frontend
            Block::append('scripts');
        }
    }
```